requirements/extra-ml-requirements.txt

```diff
@@ -10,6 +10,10 @@ fastai>=2.4.1
 spacy>=3.3.0
 # Required by mlflow.tensorflow
 tensorflow>=2.10.0
+# tensorflow-macos<=2.12.0  # Comment out the line above and uncomment this line if setting up dev
```

Comment:
Ah shoot I just realized I made a typo here, I meant for this to be `tensorflow-macos>=2.10.0`